From a66811c487ecf8ba8724879c253bb10dfa82aeb5 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Fri, 1 Jun 2018 19:57:32 +0300
Subject: [PATCH] configure_tool.sh: do not quote the argument to 'command'

As it seems to break things if the argument has spaces and arguments in it.

Upstream-Status: Pending
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 cnf/configure_tool.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cnf/configure_tool.sh b/cnf/configure_tool.sh
index 32201c0..461bc68 100644
--- a/cnf/configure_tool.sh
+++ b/cnf/configure_tool.sh
@@ -2,7 +2,7 @@
 
 tryprog() {
 	log "trying $1=$2"
-	if command -v "$2" 1>/dev/null 2>/dev/null; then
+	if command -v $2 1>/dev/null 2>/dev/null; then
 		define "$1" "$2"
 		result "$2"
 		return 0
-- 
2.17.0

